t/Test28-fregrid_stretched.sh

```diff
@@ -0,0 +1,112 @@
+               --stretch_factor 2.5 \
+               --target_lon 262.4 \
+               --target_lat 35.4 \
+               --nlon 512 \
+               --grid_name C384_grid_35.4
+  [ "$status" -eq 0 ]
+
+
+#Create stretched grid mosaic
+  run command make_solo_mosaic \
+                --num_tiles 6 \
+                --dir ./ \
+                --mosaic_name C384_mosaic_32.0 --tile_file C384_grid_32.0.tile1.nc,C384_grid_32.0.tile2.nc,C384_grid_32.0.tile3.nc,C384_grid_32.0.tile4.nc,C384_grid_32.0.tile5.nc,C384_grid_32.0.tile6.nc
+  [ "$status" -eq 0 ]
+
+  run command make_solo_mosaic \
+                --num_tiles 6 \
+                --dir ./ \
+                --mosaic_name C384_mosaic_34.0 --tile_file C384_grid_34.0.tile1.nc,C384_grid_34.0.tile2.nc,C384_grid_34.0.tile3.nc,C384_grid_34.0.tile4.nc,C384_grid_34.0.tile5.nc,C384_grid_34.0.tile6.nc
+  [ "$status" -eq 0 ]
+
+  run command make_solo_mosaic \
+                --num_tiles 6 \
+                --dir ./ \
+                --mosaic_name C384_mosaic_35.4 --tile_file C384_grid_35.4.tile1.nc,C384_grid_35.4.tile2.nc,C384_grid_35.4.tile3.nc,C384_grid_35.4.tile4.nc,C384_grid_35.4.tile5.nc,C384_grid_35.4.tile6.nc
+  [ "$status" -eq 0 ]
+
+# stretched grid lats 32.0 34.0 35.4
+  run bash -c 'fregrid \
+                --input_mosaic C384_mosaic_32.0.nc \
+                --nlon 640 \
+                --nlat 400 \
+                --latBegin 15.0 \
+                --latEnd 65.0 \
+                --lonBegin 230.0 \
+                --lonEnd 310.0 \
+                --remap_file fregrid_remap_file_640_by_400_32.0.nc \
+                --output_file out_32.0.nc --check_conserve \
+                | awk 'NR==4' | rev | cut -c39-49 | rev'
+  var_32_0=$output | awk '{ print sprintf("%.9f", $1); }'
+  echo $output
+  [[ ${var_32_0} < 0.0 ]]
+
+  run bash -c 'fregrid \
+                --input_mosaic C384_mosaic_34.0.nc \
+                --nlon 640 \
+                --nlat 400 \
+                --latBegin 15.0 \
+                --latEnd 65.0 \
+                --lonBegin 230.0 \
+                --lonEnd 310.0 \
+                --remap_file fregrid_remap_file_640_by_400_34.0.nc \
+                --output_file out_34.0.nc --check_conserve \
+                | awk 'NR==4' | rev | cut -c39-49 | rev'
+  var_34_0=$output | awk '{ print sprintf("%.9f", $1); }'
+  echo $output
+  [[ ${var_34_0} < 0.0 ]]
+
+  run bash -c 'fregrid \
+                --input_mosaic C384_mosaic_35.4.nc \
+                --nlon 640 \
+                --nlat 400 \
+                --latBegin 15.0 \
+                --latEnd 65.0 \
+                --lonBegin 230.0 \
+                --lonEnd 310.0 \
+                --remap_file fregrid_remap_file_640_by_400_35.4.nc \
+                --output_file out_35.4.nc --check_conserve \
+                | awk 'NR==4' | rev | cut -c39-49 | rev'
+  var_35_4=$output | awk '{ print sprintf("%.9f", $1); }'
+  echo $output
+  [[ ${var_35_4} < 0.0 ]]
+ 
+  cd ..
+  rm -rf Test28
+}
```

Comment:
Nice work @LuisSalbey . Is there a way for the users to see the output of these tests?

Reply:
I don't think the users can see the output if the tests pass and the Test28 and Test29 directory are deleted.

Reply:
Since we do not have an automated way to evaluate the grids yet, we need to have access to these outputs to evaluate the results. Can the outputs go to some temp space and not get deleted automatically?

Reply:
Yes. we can for the moment not remove the Test28 and Test29 at the end of the test.

Reply:
Niki and Luis, I don't think these repository tests are a good avenue for storing and making available the results of the conservation tests without more moving pieces at least. Not removing the temporary test directories will create problems for the automated testing environments and also doesn't make the conservation test results (as printed by fregrid) any more visible.

Reply:
Point taken. Do we have access  to the stdout of the test in gitub Travis CI?